js/ui/status/automaticUpdates.js

```diff
@@ -32,9 +32,11 @@ const Shell = imports.gi.Shell;
 const St = imports.gi.St;
 
 const Main = imports.ui.main;
+const MessageTray = imports.ui.messageTray;
 const PanelMenu = imports.ui.panelMenu;
 const PopupMenu = imports.ui.popupMenu;
 
+const NM_SETTING_AUTOMATIC_UPDATES_NOTIFICATION_TIME = "connection.automatic-updates-notification-time";
```

Comment:
Can we use a shorter name? Actually, this makes me wonder whether we need a different namespace, or even an additional one. I'm thinking we might need either:
```
connection.automatic-updates.latest-notification-time
connection.automatic-updates.allow-downloads
connection.automatic-updates.tariff-enabled
```

Or simply:
```
automatic-updates.latest-notification-time
automatic-updates.allow-downloads
automatic-updates.tariff-enabled
```
... but I don't know whether there other components relying on those names. What do you think?

Reply:
Or... if the `connection.` namespace actually implies "automatic updates", maybe we can simply use `connection.latest-notification-time`?

Reply:
Looks like this long name was agreed already with other people working in the metered data project. Ignore my comment then

Reply:
As I said yesterday in the meeting, and mentioned to @GeorgesStavracas already a few times, I'm fine with whatever you agreed with. I simply didn't know where this came from, but you'll be the ones using it so :+1: